packages/docprovider/src/requests.ts

```diff
@@ -36,6 +38,45 @@ export interface ISessionModel {
   sessionId: string;
 }
 
+/**
+ * Call the API extension
+ *
+ * @param endPoint API REST end point for the extension
+ * @param init Initial values for the request
+ * @returns The response body interpreted as JSON
+ */
+export async function requestAPI<T = any>(
```

Comment:
I'm not sure about the name of this function. It's basically a wrapper around `ServerConnection.makeRequest`, so should it be called `makeRequest` too?
Also, it seems that it could be used by other functions in this file, although we could do that in another PR.

Reply:
I copied this helper from other Jupyter projects

Reply:
Should it be part of JupyterLab then?

Reply:
I don't know, this function is also defined in the extension template https://github.com/jupyterlab/extension-template/blob/5696924f5b7adf07cf07ea9d3510c66914fbb770/template/src/%7B%25%20if%20kind%20%3D%3D%20'server'%20%25%7Dhandler.ts%7B%25%20endif%20%25%7D.jinja#L12